### DISCUSSION TITLE
Use git to solve Global problems.

### DISCUSSION THREAD

**Discussion by tobias-wilfert** [community/community#21923] [New to GitHub]
<p>So I got that idea: We as the human race are undergoing tuff times and in the future we will need to face even tuffer problems(famine, overpopulation, equality of chance, avoiding Nuclear war, ending excisting wars) but we are the fisrt generation to have GitHub so wouldn’t it be possible to make a repository(Earth our 127.0.0.1), then add all our issues to it and then as a opean source comunity try to find sollutions to them? Evenualty creating new branches focused on somving the issues and then merging them together in one beautiful repository (masterplan)?</p>

**Comment by DanerSound** [comment]
<p>it would be a good thing … but you need a team</p>

**Comment by ThirdEyeRose** [comment, marked as answer]
<p>I think you’re definitely onto something here, though it may be helpful to break down larger problems into smaller ones so that they can be more easily worked on.</p>
<p>Have you checked out <a href="https://github.com/collections/social-impact" rel="nofollow noopener">GitHub’s collection of Social Impact repositories</a>? If not, these are just a few Open Source projects that are actively being developed to help tackle the world’s problems. You might find one to contribute to or even come up with ideas to start your own project.</p>
